Index: src/main/java/io/u02/model/Book.java
===================================================================
--- src/main/java/io/u02/model/Book.java	(revision 2c145936bfedd51dc54a4518e04d0ad836dcc457)
+++ src/main/java/io/u02/model/Book.java	(revision 2c145936bfedd51dc54a4518e04d0ad836dcc457)
@@ -1,39 +0,0 @@
-package io.u02.model;
-
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-
-@Entity
-public class Book {
-    @Id
-    // @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private Long id;
-    private String title;
-
-    public Book() {}
-
-    public Book(Long id, String title) {
-        super();
-        this.id = id;
-        this.title = title;
-    }
-
-    public Long getId() {
-        return id;
-    }
-
-    public String getTitle() {
-        return title;
-    }
-
-    public void setId(Long id) {
-        this.id = id;
-    }
-
-    public void setTitle(String title) {
-        this.title = title;
-    }
-
-}
Index: src/main/java/io/u02/service/BookService.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/io/u02/service/BookService.java	(revision 2c145936bfedd51dc54a4518e04d0ad836dcc457)
+++ src/main/java/io/u02/service/BookService.java	(date 1584456105667)
@@ -1,6 +1,6 @@
 package io.u02.service;
 
-import io.u02.model.Book;
+import io.u02.entity.Book;
 import io.u02.repository.BookRepository;
 
 import org.springframework.beans.factory.annotation.Autowired;
Index: src/main/java/io/u02/controller/BookController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/io/u02/controller/BookController.java	(revision 2c145936bfedd51dc54a4518e04d0ad836dcc457)
+++ src/main/java/io/u02/controller/BookController.java	(date 1584455862598)
@@ -1,6 +1,6 @@
 package io.u02.controller;
 
-import io.u02.model.Book;
+import io.u02.entity.Book;
 import io.u02.service.BookService;
 
 import org.springframework.web.bind.annotation.*;
Index: src/main/java/io/u02/repository/BookRepository.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/io/u02/repository/BookRepository.java	(revision 2c145936bfedd51dc54a4518e04d0ad836dcc457)
+++ src/main/java/io/u02/repository/BookRepository.java	(date 1584455862582)
@@ -1,6 +1,6 @@
 package io.u02.repository;
 
-import io.u02.model.Book;
+import io.u02.entity.Book;
 
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
Index: pom.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- pom.xml	(revision 2c145936bfedd51dc54a4518e04d0ad836dcc457)
+++ pom.xml	(date 1584448520310)
@@ -44,6 +44,12 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
+			<version>1.18.12</version>
+			<scope>provided</scope>
+		</dependency>
 	</dependencies>
 
 	<build>
